go-controller/pkg/ovn/gress_policy.go

```diff
@@ -65,6 +64,7 @@ func (pp *portPolicy) getL4Match() (string, error) {
 }
 
 func newGressPolicy(policyType knet.PolicyType, idx int, namespace, name string) *gressPolicy {
+
```

Comment:
As Dan comments, please remove this one (if only to be consistent with the same style along the file)

Reply:
done